controllers/feeds/comment.js

```diff
@@ -56,12 +56,13 @@ module.exports = async (req, res) => {
       data: {
         feed_id: body.activity_id,
         type: 'feed'
-      }
+      },
+      token: userToken.token
     };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The token is being added to the payload before checking if `userToken` exists. This could lead to a potential error if `userToken` is undefined or null.
```diff
-       data: {
-         feed_id: body.activity_id,
-         type: 'feed'
-       },
-       token: userToken.token
-      };
-      if (userToken) {
+      if (userToken) {
+        const payload = {
+          data: {
+            feed_id: body.activity_id,
+            type: 'feed'
+          },
+          token: userToken.token
+        };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->